client/src/components/HealthCard/HealthCard.jsx

```diff
@@ -0,0 +1,219 @@
+                />
+                <Typography className={styles.title} component="h1">
+                  {issuer.title}
+                </Typography>
+              </div>
+            </Box>
+            <Box display="flex" alignItems="flex-end" className={styles.group9}>
+              <Box display="flex" flexDirection="column" alignItems="flex-start"
+                className={styles.flexCol}
+              >
+                <Typography className={styles.nameLabel}>NAME</Typography>
+                <Typography className={styles.name}>
+                  {healthCard.patient.full_name}
+                </Typography>
+              </Box>
+              <Box display="flex" flexDirection="column" alignItems="flex-start" className={styles.flexCol1}>
+                <Typography className={styles.dateOfBirthLabel}>
+                  DATE OF BIRTH
+                </Typography>
+                <Typography className={styles.dateOfBirth}>
+                  {dateOfBirthDisplay}
+                </Typography>
+              </Box>
+              <IconButton
+                className={styles.eyeOutline}
+                aria-label="toggle datofbirth visibility"
+                onClick={handleClickShowDateOfBirth}
+              >
+                <VisibilityIcon />
+              </IconButton>
+            </Box>
+          </Box>
+          <Divider className={styles.line} variant="middle" />
+          <Box display="flex" flexDirection="row" alignItems="flex-start" alignSelf="center"
+              className={styles.flexRow1}>
+            <Box
+              display="flex"
+              flexDirection="column"     
+              alignItems="flex-start"
+              className={styles.flexCol2}
+            >
+              <Box display="flex" alignItems="flex-start" className={styles.group12}>
+                <Typography className={styles.covid19Vaccination}>
+                  COVID-19 VACCINATION RECORD
+                </Typography>
+              </Box>
+              <List>
+                {healthCard.immunizations.map((item, index) => (
+                  <div key={index}>
+                    <ListItem>
+                      <HealthCardVaccination index={index} {...item} />
+                    </ListItem>
+                  </div>
+                ))}
+              </List>
+            </Box>
+            <div>
+              {qrCodesRendered.map((qrCode, index) => (
+                <Box display="flex" alignItems="center" justifyContent="flex-end"
+                    className={styles.group5} key={index}>
+                  <img className={styles.qrCode} src={qrCode} />
+                </Box>
+              ))}
+              <Typography className={styles.issSmartHealthCar} paragraph={true}>
+                SMART&reg; Health Card
+              </Typography>
+            </div>
+          </Box>
+        </CardContent>
+      </Card>
+    </Box>
+  );
+}
+
+function HealthCardVaccination(props) {
```

Comment:
Should we create a separate component for this?

Reply:
I don't think so, it's not really usable independently as a separate component.